src/proxies/groupsapi.js

```diff
@@ -0,0 +1,119 @@
+const platformClient = require('purecloud-platform-client-v2');
+const retry = require('@lifeomic/attempt').retry;
+
+let groupsMap = new Map();
```

Comment:
I would avoid using a Map for now and just use an object. Your code below does not use the Map syntax.
```suggestion
const groupsMap = {};
```